MobileCenterCrashes/MobileCenterCrashes/MSCrashes.mm

```diff
@@ -779,6 +775,7 @@ - (void)handleLatestCrashReport {
         NSURL *cacheURL = [self.crashesDir URLByAppendingPathComponent:cacheFilename];
         [crashData writeToURL:cacheURL atomically:YES];
         self.lastSessionCrashReport = [MSErrorLogFormatter errorReportFromCrashReport:report];
+        [MSWrapperExceptionManager correlateLastSavedWrapperExceptionToReport:@[self.lastSessionCrashReport]];
```

Comment:
This line brought up some design concerns that are not related to your PR. We can talk offline.